docs/xks/developer-guide/cloud-iam.md

```diff
@@ -53,6 +53,12 @@ az login --identity
 az account show
 ```
 
+#### Good practices
```

Comment:
I think the advise below is important enough to be part of the section proper. (Also, it is a bit premature to have a  heading when there is one advice.)

Reply:
Done!